ballerina/http_interceptors.bal

```diff
@@ -51,3 +53,19 @@ service class DefaultErrorInterceptor {
         return alreadyBuiltErrorResponse;
     }
 }
+
+# The class used by the runtime to invoke the `decodeJwt` method to add jwt values to the request context.
+isolated class JwtDecoder {
+
+    # Gets invoked during object initialization.
+    #
+    isolated function init() {}
```

Comment:
Can't we remove this function with empty body ?

Reply:
Removed